shell/src/main/java/alluxio/cli/fsadmin/command/ReportCommand.java

```diff
@@ -114,105 +96,101 @@ public String getCommandName() {
 
   @Override
   public int run(CommandLine cl) throws IOException {
-    try {
-      String[] args = cl.getArgs();
-
-      if (cl.hasOption(HELP_OPTION_NAME)
-          && !(args.length > 0 && args[0].equals("capacity"))) {
-        // if category is capacity, we print report capacity usage inside CapacityCommand.
-        System.out.println(getUsage());
-        System.out.println(getDescription());
-        return 0;
-      }
-
-      // Get the report category
-      Command command = Command.SUMMARY;
-      if (args.length == 1) {
-        switch (args[0]) {
-          case "capacity":
-            command = Command.CAPACITY;
-            break;
-          case "configuration":
-            command = Command.CONFIGURATION;
-            break;
-          case "metrics":
-            command = Command.METRICS;
-            break;
-          case "summary":
-            command = Command.SUMMARY;
-            break;
-          case "ufs":
-            command = Command.UFS;
-            break;
-          default:
-            System.out.println(getUsage());
-            System.out.println(getDescription());
-            throw new InvalidArgumentException("report category is invalid.");
-        }
-      }
-
-      // Only capacity category has [category args]
-      if (!command.equals(Command.CAPACITY)) {
-        if (cl.getOptions().length > 0) {
-          throw new InvalidArgumentException(
-              String.format("report %s does not support arguments: %s",
-                  command.toString().toLowerCase(), cl.getOptions()[0].getOpt()));
-        }
-      }
-
-      // Check if Alluxio master and client services are running
-      try (CloseableResource<FileSystemMasterClient> client =
-               FileSystemContext.get().acquireMasterClientResource()) {
-        MasterInquireClient inquireClient = null;
-        try {
-          InetSocketAddress address = client.get().getAddress();
-          List<InetSocketAddress> addresses = Arrays.asList(address);
-          inquireClient = new PollingMasterInquireClient(addresses, () ->
-              new ExponentialBackoffRetry(50, 100, 2));
-        } catch (UnavailableException e) {
-          System.err.println("Failed to get the leader master.");
-          System.err.println("Please check your Alluxio master status");
-          return 1;
-        }
-        try {
-          inquireClient.getPrimaryRpcAddress();
-        } catch (UnavailableException e) {
-          System.err.println("The Alluxio leader master is not currently serving requests.");
-          System.err.println("Please check your Alluxio master status");
-          return 1;
-        }
-      }
+    String[] args = cl.getArgs();
```

Comment:
the change to this method is just removing the outer `try... finally`. If we want to close our clients, that should be done in AbstractFsAdminCommand.